atlasdb-config/src/test/java/com/palantir/atlasdb/http/VersionSelectingClientsTest.java

```diff
@@ -69,4 +79,36 @@ public void drawsSeparateSamples() {
         verify(oldService, atLeast(10)).getFreshTimestamp();
         verify(newService, atLeast(10)).getFreshTimestamp();
     }
+
+    @Test
+    public void allServicesUseTheSameErrorMetric() {
```

Comment:
Hmm, I was thinking of adding support for tagging the error metric by some experiment identifier, though that might cost more money and might not be worthwhile as I assume we won't be running parallel experiments very often.

Reply:
Yeah, it would be easy to add tags, but seems unnecessarily elaborate. We can add if this turns out to be too noisy